packages/jest-config/src/validConfig.js

```diff
@@ -55,9 +55,13 @@ module.exports = ({
   modulePathIgnorePatterns: ['<rootDir>/build/'],
   modulePaths: ['/shared/vendor/modules'],
   name: 'string',
+  noDefaultReporters: false,
```

Comment:
I'm unsure about this configuration option. Alternatively we could use

```
reporters: ['summary', 'default'],
```

which gets overwritten when `reporters` is specified in a user configuration. However, I also understand that the ergonomics of that aren't great. Instead of a boolean, I'd almost do something like this:

* `reporters` – this overwrites the default in Jest.
* `additionalReporters` – this just concatenates to the default or the specified `reporters` config.

Not sure if this is necessarily better and maybe my problem is just with the name of this boolean config, though.

cc @dmitriiabramov What do you think?